shutterstock-ruby.gemspec

```diff
@@ -4,7 +4,7 @@ require 'date'
 
 Gem::Specification.new do |s|
   s.name        = 'shutterstock-ruby'
-  s.version     = '0.2.0'
+  s.version     = '0.4.0'
```

Comment:
I assume there's a `0.3.x` upstream?

Reply:
yeah, our master has diverged. If this gets merged then we can reset our master to be the same as theirs. I didn't want to just change it as I think the we app is currently pointing to here and might be looking for a certain commit